Userland/Utilities/ping.cpp

```diff
@@ -30,6 +30,10 @@ static int min_ms;
 static int max_ms;
```

Comment:
Regarding the commit title, please write commit titles in the imperative mood, for instance:
> ping: Handle optional field in the IPv4 header